giles-eco-tardis/src/main/java/edu/asu/diging/tardis/core/kafka/RequestReceiver.java

```diff
@@ -40,10 +40,11 @@ public void receiveMessage(String message) {
             return;
         }
         
-        if (request.getFileType() == FileType.PDF) {
+        if (request.getFileType() == FileType.IMAGE) {
             manager.extractImages(request);
-        }
-        logger.info("File not a pdf. Skip processing.");
-        // otherwise do nothing (we only care about pdfs)
+        } else {
+            // otherwise do nothing (we only care about image files)
+            logger.info("File not an image. Skip processing.");
```

Comment:
maybe make this `debug` or we get a lot of these messages in the logs which isn't really useful unless you are trying to debug why a file is not being processed.